### PR TITLE
Proofread metrics.md

### PR DESCRIPTION
**Pull request by MarkvanMents**

### CODE REVIEW

File: content/en/docs/refguide/runtime/metrics.md [hunk 1]
```diff
@@ -272,17 +265,17 @@ The above filter accepts metrics which starts with **"app."**, **"employee."**,
...
 
-### Notes
+### 2.3 Notes
 
 {{% alert color="info" %}}
 The following should be taken into account when configuring the metrics registries.
 {{% /alert %}}
 
-1. There are also some internal metrics used by Mendix. Filters will also have an effect on them. For example metrics emitted by Mendix which start with "mx.".
+1. Filters will also have an effect on internal metrics used by Mendix. For example, metrics emitted by Mendix which start with `mx.`.
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/runtime/metrics.md]: It is recommended to use Present tense in our style guide, Future tense is used several times in this section, maybe some  sentences can use Present?

File: content/en/docs/refguide/runtime/metrics.md [hunk 2]
```diff
@@ -39,207 +39,200 @@ Micrometer can send metrics to multiple registries. To configure micrometer for
...
-#### result
-
-The result of applying the filter. If the filter matches the criteria, the result value will be applied to the metric. Supported values are: `accept` (or) `deny`.
-
-#### values
-
-A list of values to be applied in the given filter type.
-
-##### Example 1
+#### 2.2.1 Filter Example 1
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/runtime/metrics.md]: Other examples were not in a separate section, not sure if we want to align here or it is important for users to quickly find filter examples.

File: content/en/docs/refguide/runtime/metrics.md [hunk 3]
```diff
@@ -39,207 +39,200 @@ Micrometer can send metrics to multiple registries. To configure micrometer for
...
-#### [Influx](https://www.influxdata.com/)
+#### 2.1.3 Influx{#influx}
 
-  `uri`: The URI for the Influx backend.
-  `db`: The database name to which to send the metrics.
-  `userName`: The userName for authentication.
-  `password`: The password for authentication.
-  `step`: The step size or reporting frequency to use.
-  `enabled`: set to `true` to enable the registry. This helps in switching the meter _on_ and _off_ while keeping the settings in the configuration.
+* `uri` – the URI for the Influx backend.
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/runtime/metrics.md]: back end (according to our Style guide)

**Comment by MariaShaposhnikova** [comment]
@MarkvanMents , the document look great now! I only had a few small comments.

File: content/en/docs/refguide/runtime/custom-settings/_index.md [hunk 4]
```diff
@@ -247,14 +247,15 @@ The settings below influence the behavior of the Mendix web client.
...
-## 10 Metrics Settings
+## 10 Metrics Settings{#metrics-settings}
 
 The settings below configure metrics through [micrometer](https://micrometer.io/docs). See [Metrics](/refguide/metrics/) for more information and the specification of the settings format. 
 
 | Name | Description | Default Value |
 | --- | --- | --- |
 | **Metrics.Registries** | Registries to send metrics to |   |
 | **Metrics.ApplicationTags** | Common tags used for every meter |   |
+| **com.mendix.metrics.Type** | Enable access to Micrometer metrics through Java APIs | micrometer |
```
Comment (sayyildiz) [inline content/en/docs/refguide/runtime/custom-settings/_index.md]: What about moving this to the first line may be better as it hierarchically higher than the other 2?

Comment (MarkvanMents) [reply content/en/docs/refguide/runtime/custom-settings/_index.md]: Good suggestion. Also first alphabetically so a great decision all round.

**Pull request merged**